content/terraform/v1.12.x/docs/language/values/outputs.mdx

```diff
@@ -0,0 +1,97 @@
+---
+page_title: Use outputs to expose module data
+description: Learn how to output data from a module to expose information about your infrastructure on the CLI, in HCP Terraform, and to other Terraform configurations.
+---
+
+# Use outputs to expose module data
+
+> **Hands-on:** Try the [Output data from Terraform](/terraform/tutorials/configuration-language/outputs) tutorial.
+
+Outputs let you expose information about your infrastructure on the command line, in HCP Terraform, and in other Terraform configurations. The `output` block serves the following purposes in Terraform:
+
+- Child modules can expose resource attributes to parent modules.
+- Root modules can display values in CLI output.
+- Other Terraform configurations using [remote state](/terraform/language/state/remote) can access root module outputs with the `terraform_remote_state` data source.
+- Pass information from a Terraform operation to an automation tool.
+
+## Define outputs
+
+Add `output` blocks to export information about your module. For example, you can add two `output` blocks to expose the ID and the IP address of your configuration's web server:
```

Comment:
```suggestion
Add [`output` blocks](/terraform/language/block/output) to export information about the infrastructure objects you are managing with Terraform. For example, you can add two `output` blocks to expose the ID and the IP address of your configuration's web server:
```
We should link to the reference docs for the output block. Is that a more precise description of the information the output block exports?

Reply:
You can set output values to any valid expression! I'll add this link a bit further down and will specify the value flexibility!